easybuild/easyconfigs/c/CrayPGI/CrayPGI-2016.04.eb

```diff
@@ -0,0 +1,21 @@
+easyblock = 'CrayToolchain'
+
+name = 'CrayPGI'
+version = '2016.04'
+
+homepage = 'http://www.pgroup.com/'
+description = """Toolchain using Cray compiler wrapper, using PrgEnv-pgi module."""
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+
+dependencies = [
+     # PrgEnv version is not pinned, as Cray recommends to use the latest (default) version
+     # ('PrgEnv-pgi/' + version, EXTERNAL_MODULE),
+     ('PrgEnv-pgi/5.2.82', EXTERNAL_MODULE),
```

Comment:
@jgphpc this should *not* be version pinned, see comment, just use `'PrgEnv-pgi'`